lib/store.js

```diff
@@ -9,6 +9,15 @@ function isFunc(object) {
   return (typeof object === 'function');
 }
 
+function devTools () {
+  if (typeof window === 'object'
+      && typeof window.devToolsExtension !== 'undefined') {
```

Comment:
Why wouldn't we, actually? I really kind of like the idea to have these kinds of debugging capabilities production, the more so as having them wouldn't even bloat our bundle...